definitions/ext-network_dhcp_service/giga-dhcpv4-dashboard.json

```diff
@@ -0,0 +1,195 @@
+{
+  "name": "[nrJosh] Entity Definitions - network_dhcpV4_service",
+  "description": null,
+  "pages": [
+    {
+      "name": "Individual Entity",
+      "description": null,
+      "widgets": [
+        {
+          "title": "DHCP Service Endpoint",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 4
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon select latest(dst_ip) as `Service IP`, latest(dst_port) as `Service Port`, filter(uniqueCount(dhcp_xid), where dhcp_xid is not null and dhcp_xid != '') as `DHCP Transactions` WHERE app_name = 'dhcp' AND instrumentation.provider = 'Gigamon'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "DHCP Transactions",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 4,
+            "height": 4
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon select uniqueCount(dhcp_xid) AS 'Count (unique)' WHERE app_name = 'dhcp' AND instrumentation.provider = 'Gigamon'  SINCE 1 HOUR AGO COMPARE WITH 1 HOUR AGO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
```

Comment:
If the time is fixed ignoreTimeRange should be true
Also I would add some description in the title (f.e. DHCP Transactions last hour) to make the user aware that it's definied timerange will not be used